VRDR.HTTP/Program.cs

```diff
@@ -47,57 +44,80 @@ public static string SendResponse(HttpListenerRequest request)
             string requestBody = GetBodyContent(request);
             DeathRecord deathRecord = null;
 
-            Console.WriteLine($"Request from: {request.UserHostAddress}, type: {request.ContentType}, url: {request.RawUrl}.");
+            Console.WriteLine(
+                $"Request from: {request.UserHostAddress}, type: {request.ContentType}, url: {request.RawUrl}.");
 
             // Look at content type to determine input format; be permissive in what we accept as format specification
-            switch (request.ContentType)
+            try
+            {
+                switch (request.ContentType)
+                {
+                    case string ijeType when new Regex(@"ije").IsMatch(ijeType): // application/ije
+                        IJEMortality ije = new IJEMortality(requestBody);
+                        deathRecord = ije.ToDeathRecord();
+                        break;
+                    case string nightingaleType when new Regex(@"nightingale").IsMatch(nightingaleType):
+                        deathRecord = Nightingale.FromNightingale(requestBody);
+                        break;
+                    case string jsonType when new Regex(@"json").IsMatch(jsonType): // application/fhir+json
+                    case string xmlType when new Regex(@"xml").IsMatch(xmlType): // application/fhir+xml
+                    default:
+                        deathRecord = new DeathRecord(requestBody);
+                        break;
+                }
+            }
+            catch (Exception e)
             {
-                case string ijeType when new Regex(@"ije").IsMatch(ijeType): // application/ije
-                    IJEMortality ije = new IJEMortality(requestBody);
-                    deathRecord = ije.ToDeathRecord();
-                    break;
-                case string nightingaleType when new Regex(@"nightingale").IsMatch(nightingaleType):
-                    deathRecord = Nightingale.FromNightingale(requestBody);
-                    break;
-                case string jsonType when new Regex(@"json").IsMatch(jsonType): // application/fhir+json
-                case string xmlType when new Regex(@"xml").IsMatch(xmlType): // application/fhir+xml
-                default:
-                    deathRecord = new DeathRecord(requestBody);
-                    break;
+                throw new Exception(e.Message);

```

Comment:
Given that all you're doing in the catch block is re-throwing an exception with the same message I think we can completely take out the try/catch logic.